pkg/util/testing/core.go

```diff
@@ -69,6 +70,34 @@ func CheckLatestEvent(ctx context.Context, k8sClient client.Client,
 	return false, fmt.Errorf("mismatch with the latest event: got r:%v t:%v n:%v, reg %v", item.Reason, item.Type, item.Note, item.Regarding)
 }
 
+// CheckEventRecordedAfter checks if an event identified by eventReason, eventType, eventNote was recorded after the provided time t
+func CheckEventRecordedAfter(ctx context.Context, k8sClient client.Client,
```

Comment:
Do we need the previous implementation? It doesn't seem to be used anywhere.

Reply:
+1